packages/legend-studio/src/models/protocols/pure/PureProtocolProcessorPlugin.ts

```diff
@@ -23,11 +23,17 @@ import type { V1_PureModelContextData } from './v1/model/context/V1_PureModelCon
 import type { PureModel } from '../../metamodels/pure/graph/PureModel';
 import type { Mapping } from '../../metamodels/pure/model/packageableElements/mapping/Mapping';
 import type { Runtime } from '../../metamodels/pure/model/packageableElements/runtime/Runtime';
+import type { V1_GraphTransformerContext } from './v1/transformation/pureGraph/from/V1_GraphTransformerContext';
 
 export type V1_ElementProtocolClassifierPathGetter = (
   protocol: V1_PackageableElement,
 ) => string | undefined;
 
+export type V1_ElementTransformer = (
+  metamodel: PackageableElement,
+  context: V1_GraphTransformerContext,
```

Comment:
Might be that I'm missing something, but when do you actually use the context in the transformer?

Reply:
This is more to make sure plugins/presets can access the `context` as well.